packages/cli-kit/project.json

```diff
@@ -127,6 +127,14 @@
           "command": "pnpm tsc --noEmit",
           "cwd": "packages/cli-kit"
         }
+      },
+      "refresh-documentation": {
+        "executor": "nx:run-commands",
+        "dependsOn": ["build"],
+        "options": {
+          "command": "pnpm node dist/private/node/ui/refresh-documentation.js",
```

Comment:
I'd recommend not to mix tooling and runtime code under `src` to be able to have access to the `TS ~> JS` build pipeline. We can place it in [`/bin`](https://github.com/Shopify/cli/tree/main/bin) and use [ts-node](https://www.npmjs.com/package/ts-node) to transpile and run the code:

```bash
pnpm ts-node-esm /bin/refresh-documentation.ts
```

Reply:
And with that we can drop the `dependsOn: ["build"]`

Reply:
I did go with `ts-node-esm` first but didn't want to do an extra transpilation step. I'll do as you recommended.